lib/matplotlib/backend_bases.py

```diff
@@ -1631,7 +1667,61 @@ def __init__(self, name, canvas, key, x=0, y=0, guiEvent=None):
         self.key = key
 
 
-class FigureCanvasBase(object):
+class ExpandableBase(object):
```

Comment:
Why not just a attribute in the `ContainerBase`? right now I don't see any use other than marking.